html/pfappserver/root/static.alt/src/components/pfTableSortable.vue

```diff
@@ -14,7 +14,7 @@
       class="pf-table-sortable-empty justify-content-md-center"
     >
       <b-col cols="12" md="auto">
-        <slot name="empty" v-bind="{ isLoading }">
+        <slot name="empty" v-bind:isLoading="isLoading">
```

Comment:
Use kebab-case and the `:` shortcut.

```html
<slot name="empty" :is-loading="isLoading">
```

Reply:
I personally prefer the shish taouk case with a bit of extra hummus